opnode/rollup/derive/l1_block_info.go

```diff
@@ -0,0 +1,70 @@
+package derive
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// L1BlockInfo presents the information stored in a L1Block.setL1BlockValues call
+type L1BlockInfo struct {
+	Number         uint64
+	Time           uint64
+	BaseFee        *big.Int
+	BlockHash      common.Hash
+	SequenceNumber uint64
```

Comment:
Can you add a callout that this is a property of the L2 block unlike the rest of the properties?